pkg/cli/build.go

```diff
@@ -211,11 +213,12 @@ func Build() *cobra.Command {
 	cmd.Flags().BoolVar(&debugRunner, "debug-runner", false, "when enabled, the builder pod will persist after the build succeeds or fails")
 	cmd.Flags().BoolVarP(&interactive, "interactive", "i", false, "when enabled, attaches stdin with a tty to the pod on failure")
 	cmd.Flags().BoolVar(&remove, "rm", false, "clean up intermediate artifacts (e.g. container images)")
-	cmd.Flags().BoolVar(&failOnLintWarning, "fail-on-lint-warning", false, "turns linter warnings into failures")
```

Comment:
Let's hide this flag with at TODO to drop it (and maybe log a warning if it's true) so we can update `melange` without having to fix the flags up.

Reply:
Smart smart.